### ISSUE TITLE
Allow non-shell scripts?

### ISSUE THREAD

**Issue by spikespaz** [aviaryan/utility-bash-scripts#12]
I would like to add my script, but it's written in D and compiled, not interpreted. Can I add the binary to the repository root?

https://github.com/spikespaz/clone-by-path

**Comment by spikespaz** [comment]
Maybe including a binary is a bad idea. What about a wrapper script that downloads the latest release? Would I be allowed to use superuser to put the binary in `/usr/bin`?

What if I ported the code to Python?

I see a lot of scripts using wrappers to Python modules. Is that the best route?

**Comment by missingcharacter** [comment]
Seems like python is fine https://github.com/aviaryan/utility-bash-scripts/blob/master/.github/CONTRIBUTING.md

**Comment by aviaryan** [comment]
@spikespaz Yes, python should do. And yes, binary files or downloading binaries seems like a bad idea for now. 

@missingcharacter Thanks for the input.
